public/js/actions/management.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+import $ from 'jquery';
+import * as actionTypes from 'constants/action-types';
+
+
+export function error(debugMessage) {
+  return {
+    type: actionTypes.APP_ERROR,
+    error: {debugMessage: debugMessage},
+  };
+}
+
+export function manageCards() {
```

Comment:
I don't know if it's silly but I've been following the pep8 style of two spaces between functions. I guess I'm just used to reading code like that.